aea/configurations/schemas/definitions.json

```diff
@@ -34,6 +35,54 @@
     "resource_name": {
       "type": "string",
       "pattern": "[a-zA-Z_][a-zA-Z0-9_]*"
+    },
+    "private_key_path": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "ledger",
+        "path"
+      ],
+      "properties": {
+        "ledger": {
+          "type": "string"
+        },
+        "path": {
+          "type": "string"
+        }
+      }
+    },
+    "ledger_api": {
+      "type": "object",
+      "additionalProperties": false,
+      "required": [
+        "ledger",
+        "addr",
+        "port"
+      ],
+      "properties": {
+        "ledger": {
+          "type": "string"
+        },
+        "addr": {
+          "type": "string"
+        },
+        "port": {
+          "type": "number"
+        }
+      }
+    },
+    "author": {
+      "type": "string",
+      "pattern": "[a-zA-Z0-9_]*"
```

Comment:
Author `""` is allowed in the configuration file for backward compatibility. However, it is not going to work when interacting with the registry. 

That suggests `aea scaffold` sets the `author` field according to the session kept by the CLI tool (if any, eventually).

Reply:
Ok